pdns/dnsdistdist/doh.cc

```diff
@@ -1320,7 +1320,7 @@ static void on_dnsdist(h2o_socket_t *listener, const char *err)
       continue;
     }
 
-    if (!du->tcp && du->truncated && du->response.size() > sizeof(dnsheader)) {
+    if (!du->tcp && du->truncated && du->query.size() > sizeof(dnsheader)) {
```

Comment:
Should this also take into account `du->proxyProtocolPayloadSize`?

Reply:
It should!